tests/queries/saved_queries/commands_tests.py

```diff
@@ -108,3 +120,109 @@ def test_export_query_command_key_order(self, mock_g):
             "version",
             "database_uuid",
         ]
+class TestImportSavedQueriesCommand(SupersetTestCase):
+    def test_import_v1_saved_queries(self):
+        """Test that we can import a saved query"""
+        contents = {
+            "metadata.yaml": yaml.safe_dump(saved_queries_metadata_config),
+            "databases/imported_database.yaml": yaml.safe_dump(database_config),
+            "queries/imported_query.yaml": yaml.safe_dump(saved_queries_config)
+        }
+
+        command = ImportSavedQueriesCommand(contents)
+        command.run()
+
+        saved_query = db.session.query(SavedQuery).filter_by(
+            uuid=saved_queries_config["uuid"]
+        ).one()
+        assert saved_query.schema == "public"
+        assert saved_query.sql == (
+            """
+            -- Note: Unless you save your query, 
+            these tabs will NOT persist if you clear
+            your cookies or change browsers.
+
+            SELECT * from birth_names
+            """
+        )
```

Comment:
This test will probably fail because the SQL is formatted differently (there's no line break after "your query,", eg).

Reply:
Yeah, they do. Was wondering about this actually, should I just not test the sql or is there a comment to disable the line is too long linter?

Reply:
You can disable to line it too linter, or break the line:

```python
assert saved_query == (
    "-- Note: Unless you save your query, "
    "these tabs will NOT persist if you clear "
    ...
)